packages/notifier/src/notifier.js

```diff
@@ -2,56 +2,68 @@ import { producePromise } from '@agoric/produce-promise';
 import harden from '@agoric/harden';
 
 /**
- * Makes a notifier object, which allows a service to produce a stream of update
- * promises. Each update is a record containing { currentState, updateHandle }.
- *   currentState is whatever state the service wants to publish,
+ * Produces a pair of objects, which allow a service to produce a stream of
+ * update promises. The notifier has a single method: getUpdatesSince, while the
+ * updater has two methods, resolve and updateState. getUpdateSince can be
```

Comment:
Reads better as "updateState and resolve" so that chronology goes left-to-right. See comments below.

Reply:
done